python/ray/experimental/channel/serialization_context.py

```diff
@@ -0,0 +1,52 @@
+from typing import TYPE_CHECKING, List, Optional
+
+if TYPE_CHECKING:
+    import numpy as np
+    import torch
+
+
+class _SerializationContext:
+    def __init__(self):
+        self.torch_device: Optional["torch.device"] = None
+        self.use_external_transport: bool = False
+        self.tensors: List["torch.Tensor"] = []
+
+    def set_torch_device(self, torch_device: "torch.device") -> None:
+        self.torch_device = torch_device
+
+    def reset_tensors(self, tensors: List["torch.Tensor"]) -> List["torch.Tensor"]:
```

Comment:
this is not used?

Reply:
Ah yeah it's used in the following PR, removed.